utils/mock-data-loader/api.js

```diff
@@ -138,6 +140,21 @@ const createUser = async (user) => {
   return response.data;
 };
 
+const createInitialUser = async (user) => {
+  const response = await axios({
+    method: 'post',
+    headers: {
+      'Content-Type': 'application/json',
+      'x-api-key': API_KEY,
+      Accepts: 'application/json',
+    },
+    url: `${portalApiUrl}/v1/user`,
+    data: user,
+  }).catch((err) => { console.error(`err: ${err}`); });
```

Comment:
Can the console.error please be written as:

```javascript
console.error('Unable to create initial user %s', {$err});
```

Reply:
Yeah no problem.

Just out of interest, why is the preference to use `%s` rather than the other type of string interpolation?